docs/docs/licenses/cancel-many.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 1
+---
+
+# Cancel many Licenses
+
+This method will cancel many adhoc Licenses
+
+## Code Sample
+
+```typescript
+const { Salable } = require('@salable/node-sdk');
```

Comment:
With the Node.js world moving away from CommonJS and towards ESM, I feel we should showcase ESM import syntax in our examples.

```ts
import { Salable } from '@salable/node-sdk';
```